u8timeseries/tests/test_logging.py

```diff
@@ -12,10 +13,15 @@
 
 class LoggingTestCase(unittest.TestCase):
 
+    @classmethod
+    def setUpClass(cls):
+        logging.disable(logging.NOTSET)
+
     def test_raise_log(self):
         exception_was_raised = False
         with LogCapture() as lc:
             logger = get_logger(__name__)
+            logger.handlers = [logging.NullHandler()]
```

Comment:
what is this line doing?

Reply:
It replaces our custom handler (which was created in the get_logger() function from custom_logging.py) by a handler that does not produce any output. This is done because although LocCapture prevents the root logger from printing the log message, it does not capture the log message of our custom handler. I could have also just removed our custom handler, which might be a better option. I will do this as part of the next push.

Reply:
ok, makes sense :)